### PR TITLE
More debug info for failed integration selenium tests during CI.

### PR DESCRIPTION
**Pull request by jmchilton**

### CODE REVIEW

File: .github/workflows/integration_selenium.yaml [hunk 1]
```diff
@@ -48,3 +49,8 @@ jobs:
         with:
           name: Integration Selenium test results (${{ matrix.python-version }})
           path: 'galaxy root/run_integration_tests.html'
+      - uses: actions/upload-artifact@v2
+        if: failure()
+        with:
+          name: Selenium test results (${{ matrix.python-version }}, ${{ matrix.chunk }})
```
Comment (nsoranzo) [inline .github/workflows/integration_selenium.yaml]:
```suggestion
          name: Integration Selenium debug info (${{ matrix.python-version }})
```

**Review by nsoranzo — COMMENTED** [review]
I've changed the artifact name to distinguish it from the one above, but not sure if it's the best name.

Do you want to rebase to remove the merged commits?

**Comment by jmchilton** [comment]
@OlegZharkov Are you able to see the debug information? The test you added is failing a lot of CI jobs - any clues how to improve it?

**Comment by OlegZharkov** [comment]
> @OlegZharkov Are you able to see the debug information? The test you added is failing a lot of CI jobs - any clues how to improve it?

Hi. Sorry, didn't realize. I will investigate

**Pull request merged**